src/components/Modals/UploadButtonGroup.jsx

```diff
@@ -91,27 +92,37 @@ const UploadButtonGroup = ({ file, setFile }) => {
         onChange={(e) => setFile(e.target.files[0])}
         fullWidth
         required
-        startIcon={<SearchIcon />}
+        startIcon={<FileUploadOutlinedIcon />}
       >
-        Choose file
-        <input
-          type="file"
-          hidden
-          accept=".pdf, .docx, .doc, .txt, .rtf, .gif, .png, .jpeg, .jpg, .webp"
-        />
+        Upload File
+        <input type="file" hidden accept=".pdf, .docx, .doc, .txt, .rtf" />
```

Comment:
There could be files users wish to update that are not text/pdf files, but also image files. That was why we had additional extensions like png and jpg accepted as acceptable file inputs.

Reply:
I changed this one just because the Figma design has a separate button for media files

Reply:
Think we should add the new button in too if that's the case since there would be times when people just upload images. It might look a little funky though since it's no longer 2 over 2, but 3 over 2.

Reply:
Gotcha, added the upload photos button back in and changed the layout to column for now so it doesn't look too funky. There's a unit test that counts the number of buttons rendered, so I updated it to count 3 now.

Reply:
Sounds good, thanks!